examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -38,10 +38,10 @@ void Commands::Register(const char * clusterName, commands_list commandsList)
     }
 }
 
+static PersistentStorage gStorage;
```

Comment:
This seems like it should have an issue, in that I assume something is using that storage after we return from the function (hence after shutdown).